### PR TITLE
better sample scene

### PR DESCRIPTION
**Pull request by mutmedia**

### CODE REVIEW

File: Assets/Samples/Demo/StreamingAssets/Firmware/.empty
Comment (iatenothingbutriceforthreedays) [inline Assets/Samples/Demo/StreamingAssets/Firmware/.empty:1]: Doesn't need to change now but I think we should just get rid of StreamingAssets from the demo, it isn't necessary anymore and just extra complexity

File: Packages/org.plunderludics.UnityHawk/Samples~/Demo/DoSomething.cs
Comment (iatenothingbutriceforthreedays) [inline Packages/org.plunderludics.UnityHawk/Samples~/Demo/DoSomething.cs:1]: Uhhh does this work? Don't the class name and the filename have to match?

File: Packages/org.plunderludics.UnityHawk/Samples~/Demo/DoSomething.lua
Comment (iatenothingbutriceforthreedays) [inline Packages/org.plunderludics.UnityHawk/Samples~/Demo/DoSomething.lua:1]: Nice looks good

File: Assets/Samples/Demo/StreamingAssets/Firmware/.empty
Comment (mutmedia) [reply Assets/Samples/Demo/StreamingAssets/Firmware/.empty:1]: we don't even need the firmware anymore?
thats maybe one of the finicky parts of the setupi

Comment (iatenothingbutriceforthreedays) [reply Assets/Samples/Demo/StreamingAssets/Firmware/.empty:1]: Still need firmware but it doesn't have to be in streamingassets

Comment (mutmedia) [reply Assets/Samples/Demo/StreamingAssets/Firmware/.empty:1]: perfect, deleted

**Comment by iatenothingbutriceforthreedays** [comment]
Looks good

**Pull request merged**